packages/libs/eda/src/lib/core/utils/default-axis-range.ts

```diff
@@ -12,14 +12,16 @@ export function numberDateDefaultAxisRange(
   observedMax: number | string | undefined,
   /** are we using a log scale */
   logScale?: boolean,
-  axisRangeSpec = 'Full'
+  axisRangeSpec = 'Full',
+  histogramViz: boolean = false
```

Comment:
Thank you for your comments, @bobular 👍 Actually, I tried similar thing to change the second and fourth argument to include Custom case, but somehow I didn't try your suggestion. It would probably work: will test